lib/controls.rb

```diff
@@ -0,0 +1,40 @@
+require 'set'
+require_relative './player.rb'
+
+# Controls has winning sets
+# Controls has over, create_player, make_play, wina and error methods
+class Controls
+  attr_accessor :over
+
+  def initialize
+    @over = false
+    @wins = [Set[1, 2, 3], Set[4, 5, 6], Set[7, 8, 9], Set[1, 4, 7],
+             Set[1, 5, 9], Set[2, 5, 8], Set[3, 5, 7],
+             Set[3, 6, 9]].freeze.freeze
+  end
+
+  def create_player(player1, player2)
```

Comment:
This method is still instantiating other classes. I suggest player1 and player2 objects be parameters in your initialize method for this Controls class.